lib/services/identity.go

```diff
@@ -262,9 +262,11 @@ type OIDCAuthRequest struct {
 	// CheckUser tells validator if it should expect and check user
 	CheckUser bool `json:"check_user"`
 
-	// StateToken is generated by service and is used to validate
-	// reuqest coming from
-	StateToken string `json:"state_token"`
+	// ID is request ID generated by service and is used to validate request coming from
+	ID string `json:"id"`
+
+	// State is the state of this request which holds information about user session
+	State AuthRequestSessionState `json:"state"`
```

Comment:
why create a separate struct with one field? simply add `CSRFToken` as a property.

Can you also keep `StateToken` as a property without renaming it, as far as I understand `StateToken` == `ID` in the new notation.